custom_components/ha_washdata/ws_api.py

```
@@ -6064,3 +6301,467 @@ def ws_start_playground_cycle_detail(
+            result={
+                "imported": imported,
+                "duplicates": duplicates,
+                "capped": capped,
+                "total_backfill": len(store.get_backfill_cycles()),
+            },
+        )
+        async_clear_history_import(hass, entry_id)
+    except asyncio.CancelledError:
+        reg.finish(task, state=task_registry.STATE_CANCELLED)
+        raise
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)